srcpkgs/webkit2gtk/template

```diff
@@ -29,14 +29,15 @@ makedepends="at-spi2-core-devel libjpeg-turbo-devel libpng-devel
  woff2-devel freetype-devel libopenjpeg2-devel libavif-devel
  qt5-devel libmanette-devel libwpe-devel wpebackend-fdo-devel
  libgcrypt-devel libnuspell-devel libpsl-devel $(vopt_if x11 libXt-devel)
- $(vopt_if wayland 'MesaLib-devel libxkbcommon-devel wayland-devel wayland-protocols')"
+ $(vopt_if wayland 'MesaLib-devel libxkbcommon-devel wayland-devel wayland-protocols')
+ libjxl-devel"
 depends="gst-plugins-good1"
```

Comment:
Add `_depends` like in https://github.com/void-linux/void-packages/pull/47600/files